src/components/book.js

```diff
@@ -20,29 +21,31 @@ function Book(props) {
       <div className="titleContainer">
         <p className="genre" value={category}>{category}</p>
         <p className="bookTitle">{title}</p>
-        <p className="bookAuthor">{author}</p>
+        <p className="bookAuthor">Jane Doe</p>
         <button type="button" className="remove" onClick={remove}>Remove</button>
       </div>
       <div className="graphContainer">
-        <div className="graph" />
-        <div className="percentageContainer">
+        <div className="percentageContainer" />
+        <div className="graphParagraph">
           <p className="percentage">64%</p>
           <p className="completed">Completed</p>
         </div>
       </div>
       <div className="progressContainer">
+        <div className="progressContainer" />
         <p>Current Chapter</p>
         <p>Chapter 17</p>
-        <button type="button" className="progressBtn">Update Progress</button>
+        <button type="button" className="progressBtn btn">Update Progress</button>
       </div>
     </div>
   );
 }
 
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] <b>REQUIRED:</b> You overused `eslint-disable`. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy try to fix the errors instead of ignoring them.